README.md

```diff
@@ -35,7 +35,8 @@ incubator project on [Oct 24, 2019](https://github.com/cncf/toc/pull/297).
 |                               |
 | **Optional Specifications:**  |
 | AMQP Protocol Binding         | [v1.0.2](https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/bindings/amqp-protocol-binding.md)  | [WIP](cloudevents/bindings/amqp-protocol-binding.md)       |
-| AVRO Event Format             | [v1.0.2](https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/formats/avro-format.md)             | [WIP](cloudevents/formats/avro-format.md)                  |
+| AVRO Event Format             | [v1.0.2](https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/formats/avro-format.md)             | [WIP](cloudevents/formats/avro-format.md)     
+| AVRO Turbo Event Format       | [v1.0.2](https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/formats/avro-turbo-format.md)             | [WIP](cloudevents/formats/avro-turbo-format.md)             |
```

Comment:
I think we need to leave the first column blank until we release it - the link checker isn't happy.
I guess a question is whether we bump everything to 1.0.3 to just release it under 1.0.2. Let's discuss on this week's call.